### PR TITLE
feat: add users list

### PR DESCRIPTION
**Pull request by ManuViola77**
## Links:
[Agregar al menu un listado de usuarios](https://github.com/users/ManuViola77/projects/1/views/1#:~:text=Agregar%20al%20menu%20un%20listado%20de%20usuarios)

## What & Why:
This PR adds a users list.

image

## Risks, Mitigation & Rollback:

- [ ] Safe to Revert *check this box if this PR can be reverted without incident*

### CODE REVIEW

File: src/components/common/ListRow/index.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import { Fragment } from 'react';
+import cn from 'classnames';
+
+import './styles.css';
+
+const ListRow = ({
+  key,
```
Comment (jpirazusta) [inline src/components/common/ListRow/index.js]: You don't need to get the `key` here and pass it down. React already knows when you pass it to `ListRow`

File: src/components/common/ListRow/index.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  showIcon,
+  children,
+}) => (
+  <Fragment key={key}>
+    <button
+      className={cn('list-row-container', { ...classProps, isEnabled: !classProps?.isDisabled })}
+      onClick={onClick}
+      disabled={disabled}
+    >
+      <span className="list-row-left-text">{leftText}</span>
```
Comment (jpirazusta) [inline src/components/common/ListRow/index.js:24]:
```suggestion
      {!!leftText && <span className="list-row-left-text">{leftText}</span>}
```
Don't you need to check for its existence?

File: src/components/common/ListRow/index.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+      <div className="list-row-user">
+        <img src={photo} className="list-row-photo" alt="user" />
+        <span className="list-row-name">{name}</span>
+      </div>
+      <span className="list-row-right-text">{rightText}</span>
```
Comment (jpirazusta) [inline src/components/common/ListRow/index.js:29]: Same here

File: src/components/common/ListRow/styles.css [hunk 4]
```diff
@@ -0,0 +1,71 @@
+.list-row-container {
+  align-items: center;
+  background-color: rgba(83, 141, 78, 0.5);
+  border-color: rgba(83, 141, 78, 0.5);
```
Comment (jpirazusta) [inline src/components/common/ListRow/styles.css]: Don't you use the constants?

**Review by jpirazusta — APPROVED** [review]

File: src/components/common/ListRow/index.js (hunk 2 above)
Comment (ManuViola77) [reply src/components/common/ListRow/index.js:24]: I actually want that space that occupies even though is empty. The style has flex something and leaves a space

File: src/components/common/ListRow/index.js (hunk 3 above)
Comment (ManuViola77) [reply src/components/common/ListRow/index.js:29]: same here

File: src/components/common/ListRow/index.js (hunk 2 above)
Comment (jpirazusta) [reply src/components/common/ListRow/index.js:24]: Yes, I saw it. I thought it wasn't on purpose

**Pull request merged**